angular-ui/src/app/navbar/navbar.component.html

```diff
@@ -44,8 +44,13 @@
             type="button"
             data-bs-toggle="dropdown"
             aria-expanded="false"
+<<<<<<< HEAD
```

Comment:
same here. no idea how this happened. but keep lines 51-52. don't need 47-50 and 53.